assets/js/blocks/stock-filter/edit.tsx

```diff
@@ -51,10 +65,62 @@ const Edit = ( {
 							} )
 						}
 					/>
-				</PanelBody>
-				<PanelBody
-					title={ __( 'Settings', 'woo-gutenberg-products-block' ) }
-				>
+					<ToggleGroupControl
+						label={ __(
+							'Allow selecting multiple options?',
+							'woo-gutenberg-products-block'
+						) }
+						value={ selectType || 'multiple' }
+						onChange={ ( value: string ) =>
+							setAttributes( {
+								selectType: value,
+							} )
+						}
```

Comment:
You could make this slightly more concise like so:

```suggestion
						onChange={ ( selectType ) => setAttributes( { selectType } ) }
```

Not required, but just a suggestion.

Reply:
Unfortunately `selectType` is already declared in the scope (`attributes` destructuring). I could apply the suggestion, but that means making other parts a bit less concise, so I'd stick to current version.